pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/utils/DriverUtils.java

```diff
@@ -134,6 +135,10 @@ public static String getJavaClassName(String columnDataType) {
       case "DOUBLE":
         columnsJavaClassName = Double.class.getTypeName();
         break;
+      case "BIG_DECIMAL":
```

Comment:
Should we add this to `getSQLDataType()` as well? Is it possible to be `BIGDECIMAL` if the value is from the `ResultSet`?